wordpress-mobile/WordPress-Android
# Fix padding in layer-lists on api 22

Fixes #9518

Padding defined in layer-list xmls is sometimes ignored when used as placeholder on API 22. It seems wrapping the layer-list into a selector fixes the issues. 

Note: I recommend checking the changes in AS, the GitHub diff makes it really difficult to see what has changed.

![padding](https://user-images.githubusercontent.com/2261188/55723353-27c5f100-5a09-11e9-8d1f-160e73b295f6.jpg)

To test:
I don't think it's necessary to test all the icons.
1. Open the app on an emulator/device with API 22
2. My Site
3. Make sure the unknown blog icon has correct padding
4. Blog Posts
5. Make sure the icon for in the Author filter has correct padding
6. Repeat steps 2 - 5 on api > 22

If you want to test the other icons as well, let me know and I'll try to come up with "to test" steps. (I had to download the data, turn on airplane mode, clear cache of the app -> it wasn't very straightforward)
